crypto/x509/v3_ia5.c

```diff
@@ -29,28 +29,29 @@ char *i2s_ASN1_IA5STRING(X509V3_EXT_METHOD *method, ASN1_IA5STRING *ia5)
 {
     char *tmp;
 
-    if (!ia5 || !ia5->length)
+    if (ia5 == NULL || ia5->length == 0) {
         return NULL;
-    if ((tmp = OPENSSL_malloc(ia5->length + 1)) == NULL) {
+    }
+    if ((tmp = OPENSSL_zalloc(ia5->length + 1)) == NULL) {
         X509V3err(X509V3_F_I2S_ASN1_IA5STRING, ERR_R_MALLOC_FAILURE);
         return NULL;
     }
     memcpy(tmp, ia5->data, ia5->length);
-    tmp[ia5->length] = 0;
     return tmp;
 }
 
 ASN1_IA5STRING *s2i_ASN1_IA5STRING(X509V3_EXT_METHOD *method,
                                    X509V3_CTX *ctx, const char *str)
 {
     ASN1_IA5STRING *ia5;
-    if (!str) {
+    if (str == NULL) {
         X509V3err(X509V3_F_S2I_ASN1_IA5STRING,
                   X509V3_R_INVALID_NULL_ARGUMENT);
         return NULL;
     }
-    if ((ia5 = ASN1_IA5STRING_new()) == NULL)
+    if ((ia5 = ASN1_IA5STRING_new()) == NULL) {
         goto err;
+    }
```

Comment:
No need for curved brackets here.

Reply:
I think that it is a good style to add brackets, need them or not

Reply:
It contradicts the current style, sorry.

Reply:
fixed